lms/djangoapps/course_home_api/progress/views.py

```diff
@@ -219,7 +230,7 @@ def get(self, request, *args, **kwargs):
             'completion_summary': get_course_blocks_completion_summary(course_key, student),
             'course_grade': course_grade,
             'has_scheduled_content': has_scheduled_content,
-            'section_scores': course_grade.chapter_grades.values(),
+            'section_scores': list(course_grade.chapter_grades.values()),
```

Comment:
What was it before?

Reply:
Just got a linter error about using an iterator in a non-iteration. `values()` returns a generator, not a basic data structure thing. So pylint was helpfully warning me that whoever consumes this might not expect a generator. So easy enough to cast to a list to avoid the warning.